frontend/dynamic-demo-plugin/src/utils/telemetry.ts

```diff
@@ -0,0 +1,89 @@
+const SEGMENT_KEY = '';
+
+const initSegment = () => {
+  const analytics = ((window as any).analytics = (window as any).analytics || []);
+  if (!analytics.initialize) {
+    if (analytics.invoked)
+      window.console && console.error && console.error('Segment snippet included twice.');
+    else {
+      analytics.invoked = !0;
```

Comment:
Why `!0` instead of `true`?

Reply:
Note that this segment piece is taken as a direct copy from segment.io's provided code snippet.

However since we formatted the code, we may as well update it correctly as well.

Reply:
This script is provided by segment.io, I just copied it without changing anything. I will test if we could change `!0` to `true`.